packages/components/src/link/stories/index.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Internal dependencies
+ */
+import Link from '../';
+
+export default {
+	title: 'WooCommerce Admin/components/Link',
+	component: Link,
+};
+
+export const External = () => {
+	return (
+		<Link href="https://woocommerce.com" type="external">
```

Comment:
Would be great to showcase other link types like wc-admin and wp-admin, but I'm not sure that's possible given the environment.

Reply:
Agreed. I have an idea to support the other types in Storybook... I'll give it a try in my coming PR for the QuickLinks component, since that will be making use of the other link types.